### PR TITLE
feat: bump packages + testnet fixes

### PR DESCRIPTION
**Pull request by dohaki**
Bumps to latest @across-protocol deps and fixes some bugs discovered while adding support for Lisk and Blast.

### CODE REVIEW

File: api/limits.ts [hunk 1]
```diff
@@ -163,9 +163,9 @@ const handler = async (
       ),
       Promise.all(
         fullRelayers.map((relayer) =>
-          destinationChainId === 1
+          destinationChainId === HUB_POOL_CHAIN_ID
```
[the remark this answers is not in the document]
Comment (dohaki) [reply api/limits.ts:166]: Nope, but `HUB_POOL_CHAIN_ID` is a number :)

[the remark this answers is not in the document]
Comment (james-a-morris) [reply api/limits.ts:166]: Sweet - but in the call below this we have HUB_POOL_CHAIN_ID being passed in where a string used to be passed

[the remark this answers is not in the document]
Comment (dohaki) [reply api/limits.ts:166]: Yes, `getCachedTokenBalance` can handle both: https://github.com/across-protocol/frontend-v2/blob/141b40a7a81347d08506be147a3e18fe39319400/api/_utils.ts#L756-L760

**Review by bmzig — APPROVED** [review]
LGTM

**Pull request merged**